.github/workflows/main.yaml

```
@@ -26,12 +26,12 @@ jobs:
     timeout-minutes: 10
     strategy:
       matrix:
-        node-version: [14.x]
+        node-version: [18.x, 20.x]
```

Karma only needs to run once using a recent node version:
```suggestion
        node-version: [20.x]
```